portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,42 @@
   width: 650px;
 }
 
```

Comment:
General UI comment: 
There seems to be no margin or padding on the paragraphs when I load the UI. Adding some will make your UI look cleaner imo.